app/src/main/kotlin/de/janhopp/luebeckmensawidget/WidgetConfigurationActivity.kt

```diff
@@ -0,0 +1,39 @@
+package de.janhopp.luebeckmensawidget
+
+import android.appwidget.AppWidgetManager.EXTRA_APPWIDGET_ID
+import android.appwidget.AppWidgetManager.INVALID_APPWIDGET_ID
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import de.janhopp.luebeckmensawidget.ui.config.WidgetConfigurationScreen
+import de.janhopp.luebeckmensawidget.ui.theme.MensaTheme
+
+class WidgetConfigurationActivity : ComponentActivity() {
```

Comment:
Maybe we could add a button to launch the configuration screen from the widget itself? I needed a moment to find the menu item in my launcher - and if one don't know that there are settings, I doubt that one would find it.

Reply:
Not a fan - but maybe later, as a configuration option?
I do not really see the point in it, we could make the configuration non-optional later?

Reply:
I also don't like the idea of another button, but I think we need some way to show that there is a settings menu - making it non-optional could indeed be one 🤔 Maybe we can think of an even better way until then. ^^